web/react/components/sidebar.jsx

```diff
@@ -204,10 +171,9 @@ export default class Sidebar extends React.Component {
             windowHeight: Utils.windowHeight()
         });
     }
-    updateScrollbar() {
-    }
     onChange() {
-        this.setState(this.getStateFromStores());
+        const newState = this.getStateFromStores();
+        this.setState(newState);
```

Comment:
Why the change? The assignment does not copy. Just want to make sure that this wasn't trying to do that.